### PR TITLE
feat: fix security section

### PR DESCRIPTION
**Pull request by vladbochok**

### CODE REVIEW

**Review by StanislavBreadless — DISMISSED** [review]

File: docs/dev/security/approach.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ assumption that all other parts can be byzantine–unreliable, buggy, or comprom
 
 Additionally, we believe that security by isolation can be further strengthened by adding redundancy. In zkSync, all
 transactions submitted are validated by simple execution before inclusion in the block. So if there is a vulnerability
-in our ZK circuits, the attacker would need to break through both the cryptography \*and **\***the sequencer (our
+in our ZK circuits, the attacker would need to break through both the cryptography and the sequencer (our
```
Comment (StanislavBreadless) [inline docs/dev/security/approach.md]: Are sure this change is needed? Before, it seems that is used italics for emphasize that sequencer is needed to break the system as well (i.e. breaking cryptography only is not enough)

Comment (vladbochok) [reply docs/dev/security/approach.md]: I am sure that it shouldn't be as before: 
image

Comment (vladbochok) [reply docs/dev/security/approach.md]: idk, maybe better to change it in another way

**Review by StanislavBreadless — APPROVED** [review]

**Pull request merged**